docs/blockchain/mining/mining.mdx

```diff
@@ -56,92 +53,90 @@ Recall that, in the Helium blockchain, blocks contain some number of individual
+- In a given epoch, `8,000,000` DCs are transferred across the network
+- The HNT Oracle Price is `$8.00`
+- In this scenario, the total HNT value of DC transferred in this epoch is `10 HNT`. This
+  calculation is:`(8,000,000 DC * $0.00001 / $8.00 HNT Oracle Price)`
+- These `10 HNT` would be split proportionally to the Hotspots that did the work routing packets at
   the 1:1 rate.
-- The remaining `597.6389` from the Network Data Transfer reward would be distributed ratably among
-  the Challengers, Witnesses and Challengees.
-  - `20.6457` to the Challenger group
-  - `461.5945` to the Witness group
-  - `115.3986` to the Challengee group
+- The remaining `641.0417` from the Network Data Transfer reward would be distributed by weight
+  among the Witnesses and Challengees.
+  - `128.2083` to the Challengee group => `4.73/(4.73+18.92) = 20%`
+  - `512.8333` to the Witness group => `18.92/(4.73+18.92) = 80%`
 
-**Example 2: DC Burn exceeds 35% HNT**
+**Example 2: DC Burn exceeds 37.50% HNT**
 
-- In a given epoch, `500,000,000` DCs are spent on data transfer across the network
-- The HNT Oracle Price is `$2.00`
+- In a given epoch, `800,000,000` DCs are spent on data transfer across the network
+- The HNT Oracle Price is `$8.00`
 
-In this scenario, total HNT value of DC transferred in this epoch is `2500HNT`. This calculation
-is:`(500,000,000 DC * $0.00001 / $2 HNT Oracle Price)`
+In this scenario, the total HNT value of DC transferred in this epoch is `1000 HNT`. This
+calculation is:`(800,000,000 DC * $0.00001 / $8.00 HNT Oracle Price)`
 
-- Because the Network DC burn exceeded the `607.6389` available to the Network Data Transfer reward,
-  all Hotspots who did the work over this epoch with split the full `1113.0145HNT` proportionally.
+- Because the Network DC burn exceeded the `651.0417` available to the Network Data Transfer reward,
+  all Hotspots that did the work over this epoch will split the full `1000 HNT` proportionally.
 
 **Additional Notes on Reward Types and Payouts**
 
 - All Validators in the `Consensus` group will earn an equal reward.
-- All Hotspots participating in PoC, including `Challengers`, `Challengees` and `Witnesses` will
-  earn rewards proportional to how many events they participated in out of the total number of
-  events per epoch.
+- All Hotspots participating in PoC, including `Challengees` and `Witnesses` will earn rewards
+  proportional to how many events they participated in out of the total number of events per epoch.
 - All Hotspots participating in `Network Data Transfer` will earn rewards proportional to their
   share of the total data transfer in that epoch, as shown above.
 - Hotspots can earn one or more reward types during any given epoch.
-- Hotspots are only eligible to submit one Proof of Coverage Challenge - which results in them
-  earning the `PoC Challenger` reward - **up to once per 360 blocks**.
```

Comment:
add a description of the new targeting / poc keys structure introduced in HIP 55